src/runcode-func.jl

```diff
@@ -305,12 +305,8 @@ function affect!(
     erupt_saved::EruptSaved{Float64},
 )
     composition = param.composition
-    storeTime = param_saved_var.storeTime
-    storeTemp = param_saved_var.storeTemp
-    storeTemp = storeTemp[storeTime .< int.t]
-    storeTime = storeTime[storeTime .< int.t]
-    param_saved_var.storeTime = storeTime
-    param_saved_var.storeTemp = storeTemp
+    param_saved_var.storeTemp = param_saved_var.storeTemp[param_saved_var.storeTime .< int.t]
+    param_saved_var.storeTime = param_saved_var.storeTime[param_saved_var.storeTime .< int.t]
```

Comment:
Evaluate `param_saved_var.storeTime .< int.t` once? What is a good name for this check, `IsStoreTimeLTt`?